packages/ariakit-test/src/global.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="@testing-library/jest-dom/vitest" />
```

Comment:
Can this file be placed outside the `src` folder? Every file in the `src` folder not prefixed with `__` will automatically be included in the package exports.